src/components/Articles/NewArticleForm.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Loader from 'react-loader';
+import PropTypes from 'prop-types';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+
+
```

Comment:
Kindly consider removing this redundant line.

Reply:
@kakaemma Redundant line has been removed